platform-tests/src/api/api_test.go

```diff
@@ -0,0 +1,130 @@
+package api
+
+import (
+	"fmt"
+	"net/url"
+	"os"
+	"time"
+
+	"github.com/cloudfoundry-community/go-cfclient"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+func lg(things ...interface{}) {
+	fmt.Fprintln(GinkgoWriter, things...)
+}
+
+var _ = Describe("API Availability Monitoring", func() {
+
+	It("should have uninterupted access to cloudfoundry api during deploy", func() {
+
+		cfg := &cfclient.Config{
+			ApiAddress:        fmt.Sprintf("https://api.%s", os.Getenv("SYSTEM_DNS_ZONE_NAME")),
+			Username:          os.Getenv("CF_USER"),
+			Password:          os.Getenv("CF_PASS"),
+			SkipSslValidation: os.Getenv("SKIP_SSL_VALIDATION") == "true",
+		}
+		Expect(cfg.ApiAddress).ToNot(BeEmpty(), "SYSTEM_DNS_ZONE_NAME environment variable must be set")
+		Expect(cfg.Username).ToNot(BeEmpty(), "CF_USER environment variable must be set")
+		Expect(cfg.Password).ToNot(BeEmpty(), "CF_PASS environment variable must be set")
+
+		deployment := &Deployment{
+			AtcAddress: os.Getenv("CONCOURSE_ATC_URL"),
+			Password:   os.Getenv("CONCOURSE_ATC_PASSWORD"),
+			Username:   os.Getenv("CONCOURSE_ATC_USERNAME"),
+			Version:    os.Getenv("PIPELINE_TRIGGER_VERSION"),
+			Team:       "main",
+		}
+		Expect(deployment.AtcAddress).ToNot(Equal(""), "CONCOURSE_ATC_URL environment variable must be set")
```

Comment:
Being interested, is there a reason you used `Equal("")` whilst above you had `BeEmpty()`?